mlrun/api/api/endpoints/functions.py

```diff
@@ -189,6 +189,7 @@ async def build_function(
         with_mlrun = strtobool(data.get("with_mlrun", "on"))
     skip_deployed = data.get("skip_deployed", False)
     mlrun_version_specifier = data.get("mlrun_version_specifier")
+    client_version = data.headers.get("x-mlrun-client-version")
```

Comment:
Not going to work, search for `Header` in our code, FastAPI has a nice thing for it